src/RepoMerger.js

```diff
@@ -96,6 +96,8 @@ class RepoMerger {
 
     _plan(ms) {
         assert(ms > 0);
+        const maxMs = Math.pow(2, 31) - 1;
```

Comment:
I would add a comment to clarify why we are restricting this value. For example, `obey nodejs setTimeout() limits`.